shimmingtoolbox/load_nifti.py

```diff
@@ -0,0 +1,168 @@
+#!usr/bin/env python3
+# -*- coding: utf-8
+
+import os
+import logging
+import numpy as np
+import nibabel as nib
+import json
+import math
+
+logger = logging.getLogger(__name__)
+PHASE_SCALING_SIEMENS = 4096
+
+
+def load_nifti(path_data, modality='phase'):
+    """
+    Load data from a NIFTI type file with nibabel.
+    Args:
+        path_data (str): Absolute or relative path to the directory the acquisition data
+        modality (str): Modality to read nifti (can be phase or magnitude)
+    Returns:
+        nibabel.Nifti1Image.Header: List containing all information from every Nifti image
+        dict: List containing all information in JSON format from every Nifti image
+        numpy.ndarray: 5D array of all acquisition in time (x, y, z, echo, volume)
+
+    Note:
+        If 'path' is a folder containing niftis, directly output niftis. It 'path' is a folder containing acquisitions,
```

Comment:
"folder containing acquisitions" --> what do you mean by "acquisitions"? do you mean DICOM files?

Reply:
You can specify a directory wich contains multiple other directories with the data files. It just tells you that you need to choose one of the folders contained in the given path. Should I rename it? It was named like this in the MATLAB version.